src/Symfony/Component/VarDumper/Caster/ReflectionCaster.php

```diff
@@ -116,8 +116,11 @@ public static function castType(\ReflectionType $c, array $a, Stub $stub, bool $
 
     public static function castAttribute(\ReflectionAttribute $c, array $a, Stub $stub, bool $isNested)
     {
+        if (!isset($a['name'])) {
+            $a['name'] = $c->getName();
```

Comment:
but actually this would be lying on PHP < 8.4, this property doesn't exist before, isn't it?

Reply:
Yes, this is true. I'm sure how to deal with this: either make it non-virtual for on 8.3 and lower (which indeed is a "lie"), or tweaking expected output of tests to accept both virtual and real property. 🤔